eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -179,21 +179,26 @@ impl Eth2NearRelay {
     }
 
     fn get_last_slot(&mut self) -> Result<u64, Box<dyn Error>> {
-        trace!(target: "relay", "= Search for last slot on near =");
+        debug!(target: "relay", "= Search for last slot on near =");
 
         let mut slot = self.eth_client_contract.get_last_submitted_slot();
 
         let finalized_slot = self.eth_client_contract.get_finalized_beacon_block_slot()?;
-        trace!(target: "relay", "Finalized slot on near={}", finalized_slot);
+        debug!(target: "relay", "Finalized slot on near={}", finalized_slot);
 
         slot = max(finalized_slot, slot);
```

Comment:
Btw, this piece of code could be a little bit improved.

```suggestion
        let finalized_slot = self.eth_client_contract.get_finalized_beacon_block_slot()?;
        debug!(target: "relay", "Finalized slot on near={}", finalized_slot);

        let last_submitted_slot = self.eth_client_contract.get_last_submitted_slot();
        debug!(target: "relay", "Last submitted slot={}", last_submitted_slot);

        let slot = max(finalized_slot, last_submitted_slot);
```

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/775/commits/a2077433d78c60542ef28d456c88ac578754038c